charts/brigade-dockerhub-gateway/values.yaml

```diff
@@ -0,0 +1,130 @@
+## Default values for brigade-dockerhub-gateway
+## This is a YAML-formatted file.
+## Declare variables to be passed into your templates.
+
+replicas: 1
+
+## Host should be set to the public IP address or DNS hostname for this
+## gateway. Whenever possible, it should be set accurately for a variety of
+## reasons. If applicable, the value is used both in automatic certificate
+## generation and, also if applicable, the creation of ingress routing rules
+## (advanced). Keeping the default value or otherwise setting this
+## inaccurately may result in invalid routing rules or untrusted certificates
+## (untrusted beyond merely having been self-signed).
+host: cr-gateway.example.com
```

Comment:
nit: maybe update to `dockerhub-gateway.example.com`?

Reply:
Yup. This is something I missed after realizing that I couldn't really make a single "container registry" gateway that supported Docker Hub _and_ ACR. Will fix.